tests/integration/delivery/delivery_suite_test.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2021 VMware
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package delivery_test
+
+import (
+	"context"
+	"io"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/storage/names"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/vmware-tanzu/cartographer/tests/helpers"
+
+	"github.com/vmware-tanzu/cartographer/pkg/apis/v1alpha1"
+	"github.com/vmware-tanzu/cartographer/pkg/root"
+)
+
+func TestDeliveryIntegration(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Delivery Integration Suite")
+}
+
+var (
+	testEnv          *envtest.Environment
+	c                client.Client
+	testNS           string
+	workingDir       string
+	cancel           context.CancelFunc
+	controllerError  chan error
+	controller       *root.Command
+	controllerBuffer *gbytes.Buffer
+)
+
+const DebugControlPlane = false
+
+var _ = BeforeSuite(func() {
+	var err error
```

Comment:
dang, so sad we have to have the same setup one again 😅  it'd be great if we could soon get around having a more approachable suite that lets us not have to repeat this (I don't think we necessarily have to address is in this PR though, just voicing a thought :p )